### ISSUE TITLE
Visual Studio Code Version 1.56.0 breaks terminal pane transparency

### ISSUE THREAD

**Issue by mkanet** [EYHN/vscode-vibrancy#100]
VSCode 1.56.0 added terminal tabs.  This completely broke the transparency effect in the terminal pane.  Looks so ugly.

Is there a way to fix this?
![Screenshot 05-05-2021 at 04 50 37 PM](https://user-images.githubusercontent.com/811008/117223544-c082f280-adc2-11eb-87a9-a783edc90539.png)

I had to revert back to the previous VSCode version.

**Comment by BIKA-C** [comment]
The original setting `"terminal.integrated.rendererType": "dom"` is deprecated. I do not know if there is an alternative yet.

**Comment by BIKA-C** [comment]
So, I guess the readme.md will be updated very soon

**Comment by mkanet** [comment]
Thank you.  A couple of questions.  

1. Is `"terminal.integrated.gpuAcceleration": "off"` exactly the same as `"terminal.integrated.rendererType": "dom"` in terms of performance and functionality?
2. I have an VSCode extension that also utilizes Vibrancy code.  If I wanted to automatically add the setting `"terminal.integrated.gpuAcceleration": "off"` to settings.json when my extension installs, can I add the below code to my `package.json`?  If not, how can I do that without having to manually add that setting?
```
"configuration": {
	"title": "Vibrancy Effect",
	"properties": {
		"terminal.integrated.gpuAcceleration": {
			"type": "string",
			"default": "off"
		}
	}
}
```

**Comment by Vbner** [comment]
> The original setting `"terminal.integrated.rendererType": "dom"` is deprecated.
> Use this instead: `"terminal.integrated.gpuAcceleration": "off"`
> Here is the [reference](https://code.visualstudio.com/docs/editor/integrated-terminal#_changing-how-the-terminal-is-rendered)

Thank you for your help, Appreciate it !!

**Comment by EYHN** [comment]
Thanks for the heads up guys, I've fixed this in 1.0.15. Please make sure you have upgraded to the latest version.
